sdk/keyvault/ci.yml

```diff
@@ -39,5 +39,7 @@ stages:
       Artifacts:
         - Name: azure-security-keyvault-common
           Path: azure-security-keyvault-common
+          VcpkgPortName: azure-security-keyvault-common-cpp
```

Comment:
.NET calls this Core. Did we already discuss C++ sticking to `common`?
https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault/Microsoft.Azure.KeyVault.Core